docs/docsite/rst/intro_windows.rst

```diff
@@ -312,37 +312,46 @@ What modules are available
 Most of the Ansible modules in core Ansible are written for a combination of Linux/Unix machines and arbitrary web services, though there are various
 Windows-only modules. These are listed in the `"windows" subcategory of the Ansible module index <http://docs.ansible.com/list_of_windows_modules.html>`_.
 
-In addition, the following core modules work with Windows::
-
-    fetch
-    raw
-    script
-    slurp
-    template
-    add_host
-    assert
-    debug
-    fail
-    group_by
-    include_vars
-    meta
-    pause
-    set_fact
-
+In addition, the following core modules/action-plugins work with Windows:
+
+* add_host
+* assemble
```

Comment:
Oops assemble has crept back into this list.  Maybe add ' with 'delegate_to: localhost' ... or just leave it for the paragraph after to explain.